### ISSUE TITLE
Intramural cystic spaces in gallbladder

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10045]
**Preferred term label:**
Intramural cystic spaces in gallbladder

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0005264

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Adenomyomatosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
is it certain that this feature is different from 
Rokitansky-Aschoff gallbladder sinuses
https://github.com/obophenotype/human-phenotype-ontology/issues/10044
If yes, can you provide a definition? From my reading of https://pubmed.ncbi.nlm.nih.gov/31984708/ it is hard to differentiate the two.

**Comment by MickeySegal** [comment]
I agree - that is my impression from reading between the lines at:
https://www.ncbi.nlm.nih.gov/books/NBK482244/ 
"The characteristic findings of adenomyomatosis are gallbladder wall thickening, Rokitansky-Aschoff sinuses, and the calculi or cholesterol crystals often found in sinuses. On ultrasound scanning, adenomyomatosis is seen as gallbladder wall thickening, intramural cystic spaces, and echogenic intramural foci (cholesterol crystals in the Rokitansky-Aschoff sinuses) that show comet tail artifacts. "

I'll delete this finding.

**Comment by pnrobinson** [comment]
+1 closing
